tests/e2e/users/add-user.specs.js

```diff
@@ -7,8 +7,13 @@ const commonElements = require('../../page-objects/common/common.po');
 const addedUser = 'fulltester' + new Date().getTime();
 const fullName = 'Bede Ngaruko';
 
-
-describe('Add user  : ', () => {
+/**
+ * Test Case Migrated to Wdio: add-user.wdio-spec.js
+ * Keeping Protractor tests with Skip option (prefixing "x")
+ * Once entire migration is done, come back and remove all protractor tests
+ * This helps in identifying which all tests have been migrated
+ */
+xdescribe('Add user  : ', () => {
```

Comment:
I see the comment here, but this means we have only migrated _some_ of these tests, but have stopped executing _all_ of these tests. Please do not disable all of them. Only disable the ones that have been migrated... but then since those are migrated, there isn't even a reason to keep them, so instead of disabling those specific tests, just remove them.

Reply:
This entire test class is migrated. I just skipped instead of deleting this class to keep track of what all is migrated .I can remove if that 's what we prefer.

Reply:
Yes, please. Always remove unused code; we can get it back from git we if ever need it later.

Reply:
Done